pyccel/ast/utilities.py

```diff
@@ -78,12 +80,15 @@ def builtin_function(expr, args=None):
 decorators_mod = Module('decorators',(),
         funcs = [PyccelFunctionDef(d, PyccelInternalFunction) for d in pyccel_decorators.__all__])
 pyccel_mod = Module('pyccel',(),(),
-        imports = [Import('decorators', decorators_mod)])
+        imports = [Import('decorators', decorators_mod),
+                    Import('cuda', AsName(cuda_mod,'cuda')),
```

Comment:
I don't think the `AsName` is needed here

Reply:
yes it is not needed.
is there a difference between this and the imports in `builtin_import_registery`?
[here](https://github.com/pyccel/pyccel-cuda/blob/ba35a691d261c91d34e464715be20c03a13d7117/pyccel/ast/utilities.py#L88)

Reply:
[c7222ff](https://github.com/pyccel/pyccel-cuda/pull/12/commits/c7222ff4882df1901dffbd1b1d82caa7a4eef674)

Reply:
Ah! This is my fault :facepalm: . The second argument to an `Import` has to be either a `Module` or an `AsName`. I obviously forgot when I was writing `builtin_import_registery`

Reply:
Fixed https://github.com/pyccel/pyccel/pull/1364